pkg/bff/server.go

```diff
@@ -349,6 +349,7 @@ func (s *Server) setupRoutes() (err error) {
 
 		// Authenticated routes
 		v1.POST("/collaborators", auth.DoubleCookie(), auth.Authorize("update:collaborators"), s.AddCollaborator)
+		v1.PUT("/collaborators", auth.DoubleCookie(), auth.Authorize("update:collaborators"), s.ReplaceCollaborator)
```

Comment:
Having a `POST` and ` PUT` to the same endpoint strikes me as a little weird; it's usually `PUT /collection/:id` -- but unless we have an ID for collaborators, this may just have to be the way it is. Can you check in with @masskoder to see if he thinks this makes sense?